iidm/iidm-impl/src/test/groovy/com/powsybl/iidm/network/impl/IdentifiableExtensionGroovyTest.groovy

```diff
@@ -44,10 +44,10 @@ class IdentifiableExtensionGroovyTest {
     @Test
     void testProperty() {
         assertFalse(s.hasProperty())
-        assertNull(s.greeting)
-        s.greeting = "hello"
```

Comment:
Your change should not impact this line of code and Groovy automatic property creation should continue to work as in the test.

Reply:
Without changes, the test failed with following message:
No signature of method: java.util.LinkedHashMap.getProperty() is applicable for argument types: (java.lang.String) values: [greeting]